src/main/java/com/example/tournamentMaker/team/TeamService.java

```diff
@@ -47,7 +47,7 @@ void createTeam(TeamRequest teamRequest) {
         });
     }
 
-    private static FootballStatistics createFootballStatistics(Team team) {
+    private FootballStatistics createFootballStatistics(Team team) {
         return new FootballStatistics(team, 0, 0, 0, 0, 0);
```

Comment:
moze lepiej przemyslec konstruktor bezparametrowy ktory by robil takie rzeczy. Albo pomyslec o metodzie ktora to robi bo glupio to wyglada